metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -1131,7 +1132,15 @@ def from_api(  # noqa: C901
                 logger.warning(
                     f"Failed to extract explore {explore_name} from model {model}: {e}"
                 )
-
+        except DeserializeError as e:
+            logger.warning(
+                f"Failed to extract explore {explore_name} from model {model}, Error: {e}"
+            )
+            reporter.report_warning(
```

Comment:
this will automatically also generate a log message

```suggestion
            reporter.warning(
```